packages/web-components/src/components/va-additional-info/va-additional-info.css

```diff
@@ -2,6 +2,13 @@
 :host {
   display: block;
 }
+
+:host([disable-border='true']) a[aria-expanded='true'] ~ #info {
```

Comment:
What about situations when `true` isn't explicitly passed, but the prop is just truthy? That's [a valid boolean attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#boolean_attributes):


> HTML5 defines restrictions on the allowed values of boolean attributes: If the attribute is present, **its value must either be the empty string** (equivalently, the attribute may have an unassigned value), or a value that is an ASCII case-insensitive match for the attribute's canonical name

Here's [an example](https://staging.va.gov/kansas-city-health-care/locations/shawnee-va-clinic/index.html):

![image](https://user-images.githubusercontent.com/2008881/159582415-ff88d790-4800-478d-aec7-e9338d16d1a8.png)

Reply:
@bkjohnson good call I added `:host([disable-border='']) a[aria-expanded='true'] ~ #info` to account for that so we are now covered for that use case